bitnami/kubeapps/templates/tests/test-tiller-proxy.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ .Release.Name }}-tiller-proxy-test"
+  annotations:
+    "helm.sh/hook": test-success
+spec:
+  containers:
+  - name: {{ .Release.Name }}-tiller-proxy-test
+    image: {{ .Values.testImage.repository }}:{{ .Values.testImage.tag }}
+    env:
+    - name: TILLER_PROXY_HOST
+      value: {{ template "kubeapps.tiller-proxy.fullname" . }}.{{ .Release.Namespace }}
+    - name: TILLER_PROXY_PORT
+      value: "{{ .Values.tillerProxy.service.port }}"
+    - name: KUBEAPPS_RELEASE
+      value: {{ .Release.Name }}
+    command: 
+    - sh
+    - -c
+    - "curl -ik -H \"Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)\" $TILLER_PROXY_HOST:$TILLER_PROXY_PORT/v1/releases | grep $KUBEAPPS_RELEASE"
```

Comment:
This is cool but I would be careful not to transform these tests into integration tests by verifying that a specific values are returned via the API and so on.

Ideally, I would just check the /healthz endpoint, which I just noticed that we do not have. Just created an issue to tackle that https://github.com/kubeapps/kubeapps/issues/502

Reply:
@migmartri in fact my idea is to use these tests as integration tests for the e2e scenario. See the description here: https://github.com/kubeapps/kubeapps/pull/442. If that goes against the test policy of helm charts we can modify it.